### ISSUE TITLE
Добавить кнопку индексации в каджый пост

### ISSUE THREAD

**Issue by mihdan** [mihdan/recrawler#9]

**Comment by standzzen** [comment]
Да, очень не хватает индивидуальной отправки постов в индекс с редактора, тк у нас стоит плагин пермалинков и для некоторых постов генерятся индивидуальные линки и если публикуешь пост то в индекс отправляется неправильный урл.. Поэтому пришлось отключить индексацию для некоторый постов.

**Comment by mihdan** [comment]
@standzzen а какой плагин пермалинков используете? Сделаем под него интеграцию

**Comment by standzzen** [comment]
> [@standzzen](https://github.com/standzzen) а какой плагин пермалинков используете? Сделаем под него интеграцию

Permalink Manager Pro
https://permalinkmanager.pro/

**Comment by standzzen** [comment]
Добавлю инфы. 
Проблема именно со стандартными постами. 
В настройках пермалинка этого плагина для обычных постов структура: /blog/категория/пост_слаг
А в индекс уходит /пост_слаг

В настройках вордперсса в пермалинках стоит Post name 
Эту настройку не трогаю, тк есть другие тайпы постов через ACF чтобы не поломать структуру там, тк если в стандарте настраиваю /блог/категория/слаг - то ломаются пермалинки для других тайпов.
